### PR TITLE
feat(ui5-combobox): implement accessibility spec

### PR DESCRIPTION
**Pull request by ivoplashkov**
FIXES:#1278

### CODE REVIEW

File: packages/main/src/ComboBox.hbs [hunk 1]
```diff
@@ -29,6 +34,7 @@
 			?pressed="{{_iconPressed}}"
 			@click="{{_arrowClick}}"
 			dir="{{dir}}"
+			accessibleName="Select Options"
```
Comment (vladitasev) [inline packages/main/src/ComboBox.hbs:37]: This should be translated

File: packages/main/src/ComboBox.js [hunk 2]
```diff
@@ -490,6 +493,32 @@ class ComboBox extends UI5Element {
 		return !this.readonly;
 	}
 
+	get accInfo() {
+		return {
+			"role": "combobox",
```
Comment (vladitasev) [inline packages/main/src/ComboBox.js:498]: Are these overwritten from somewhere (maybe multicombobox?) If yes, it's fine, if not -please hardcode them in the template. No need for variable parts for static content.

File: packages/main/src/ComboBoxPopover.hbs [hunk 3]
```diff
@@ -2,6 +2,7 @@
 	no-arrow
 	content-only-on-desktop
 	_disable-initial-focus
+	id="{{_id}}-responsive-popover"
```
Comment (vladitasev) [inline packages/main/src/ComboBoxPopover.hbs:5]: This is in a separate shadow DOM tree, won't work. You can't reference it from within the shadow tree of the main component

File: packages/main/src/ComboBox.js (hunk 2 above)
Comment (elenastoyanovaa) [reply packages/main/src/ComboBox.js:498]: You are right. We discussed to remove it.

File: packages/main/src/ComboBoxPopover.hbs (hunk 3 above)
Comment (elenastoyanovaa) [reply packages/main/src/ComboBoxPopover.hbs:5]: You are right. We will remove the aria-controls

File: packages/main/src/ComboBox.js (hunk 2 above)
Comment (ivoplashkov) [reply packages/main/src/ComboBox.js]: Done.

File: packages/main/src/ComboBox.hbs (hunk 1 above)
Comment (ivoplashkov) [reply packages/main/src/ComboBox.hbs]: Done.

**Pull request merged**